pkg/cloudprovider/providers/gce/gce_address_manager_test.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"testing"
+
+	computebeta "google.golang.org/api/compute/v0.beta"
+	compute "google.golang.org/api/compute/v1"
+)
```

Comment:
Would you consider using `github.com/stretchr/testify/assert` (for `Errorf`) and `github.com/stretchr/testify/require` (for `Fatalf`). They tend to make the code more succinct .

Reply:
I'm a fan of testify/assert for personal projects, but haven't seen enormous use of it here in K8s. Happy to use it unless someone says otherwise.